arm/Microsoft.ManagedIdentity/userAssignedIdentities/readme.md

```diff
@@ -1,6 +1,6 @@
 # User Assigned Identities `[Microsoft.ManagedIdentity/userAssignedIdentities]`
 
-This module deploys User Assigned Identities, with resource lock.
+This module deploys a user assigned identity, with resource lock.
```

Comment:
I'd propose to use the lock, or? Is it deployed by default?

Reply:
done